_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -10876,6 +10876,17 @@
 	icon_state = "darkredcorners"
 	},
 /area/security/prison/cell_block/A)
+"axO" = (
+/obj/structure/cable{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2";
+	tag = ""
+	},
+/turf/simulated/floor/plasteel{
+	icon_state = "redcorner"
+	},
+/area/security/processing)
```

Comment:
Something has fucked the keys up in this map with random moves, and a random diff off in the distance in medmaint.

Requesting you copy your edit to a temp file, grab the latest version of cyberiad, and then edit from there.